### PR TITLE
PE-1498 Upload Fail Dialog

### PR DESCRIPTION
**Pull request by javdhu**
Improves error handling for uploads.

### CODE REVIEW

File: lib/l10n/app_en.arb [hunk 1]
```diff
@@ -1355,5 +1355,13 @@
...
   "@infernoIsInFullSwing": {
     "description": "Text inviting the users to go read the Inferno rules"
+  },
+  "uploadFailed": "Upload Failed",
+  "@uploadFailed": {
+    "description": "Upload Failed dialog title"
+  },
+  "yourUploadFailed": "Upload Failed",
+  "@yourUploadFailed": {
+    "description": "Your upload failed. You have not been charged for the failed attempt."
```
Comment (matibat) [inline lib/l10n/app_en.arb:1365]: Is the description and the actual wording toggled here?

**Review by matibat — COMMENTED** [review]
I think there's one single mistake in the ARB file

File: lib/blocs/upload/upload_handles/bundle_upload_handle.dart [hunk 2]
```diff
@@ -1,117 +1,117 @@
...
+      bundle.blob,
+      wallet,
+    );
+
+    await pstService.addCommunityTipToTx(bundleTx);
+
+    await bundleTx.sign(wallet);
+
+    // Write entities to database
+    for (var folder in folderDataItemUploadHandles) {
```
Comment (karlprieb) [inline lib/blocs/upload/upload_handles/bundle_upload_handle.dart:69]: What's the difference between for and forEach in this case?

File: lib/l10n/app_en.arb (hunk 1 above)
Comment (karlprieb) [reply lib/l10n/app_en.arb:1365]: I believe these are new keys, right?

**Review by karlprieb — APPROVED** [review]

**Review by matibat — DISMISSED** [review]
We are gonna move forward with this as it is just "to have something".
Then we are fully address the pending case (partial failed upload) in a following up Story which:
- Handles the partial upload
- Applies the wording specified at the spec

_**Note** we are gonna go with English only for now because the actual wording is gonna change_

**Review by thiagocarvalhodev — DISMISSED** [review]

**Review by thiagocarvalhodev — APPROVED** [review]

**Pull request merged**